pkg/internal/checkup/trex/client.go

```diff
@@ -85,11 +85,11 @@ func NewClient(vmiSerialClient vmiSerialConsoleClient,
 
 func (c Client) StartServer() error {
 	command := "systemctl start " + SystemdUnitFileName
-	_, err := console.SafeExpectBatchWithResponse(c.vmiSerialClient, c.namespace, c.vmiName,
-		[]expect.Batcher{
-			&expect.BSnd{S: command + "\n"},
-			&expect.BExp{R: shellPrompt},
-		},
+	trafficGenConsole := console.NewExpecter(c.vmiSerialClient, c.namespace, c.vmiName)
```

Comment:
I find this name confusing with the existing `trexClient` variable that we have in executor.
Could you please rename this variable?

Reply:
changed to `trafficGenConsoleExpecter`